<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>listTickets</class>
 <widget class="QDialog" name="listTickets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="searchLineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>721</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QComboBox" name="ticket_comboBox">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>30</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="ticket_tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>851</width>
     <height>591</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
